l2-contracts/contracts/dev-contracts/TimestampAsserter.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+interface ITimestampAsserter {
+    function assertTimestampInRange(uint256 start, uint256 end) external view;
+}
+
+contract TimestampAsserter is ITimestampAsserter {
```

Comment:
Move the contract from `dev-contracts` folder, it is not dev contract if it will be used in prod

Reply:
Fixed